playbooks/roles/foreman_libvirt/tasks/libvirt.yml

```diff
@@ -1,4 +1,9 @@
 ---
```

Comment:
Is this libvirt_setup specific enough to how Foreman would want libvirt configured vs it being generically a libvirt configuration role?